packages/host/app/services/command-service.ts

```diff
@@ -31,6 +37,27 @@ export default class CommandService extends Service {
             relationships: payload.relationships,
           },
         );
+      } else if (command.name === 'searchCard') {
+        if (!hasSearchData(payload)) {
+          throw new Error(
+            "Search command can't run because it doesn't have all the arguments returned by open ai",
+          );
+        }
+        //If ai returns a relative module path, we make it absolute based upon the card_id that exists
+        let maybeCodeRef = codeRefWithAbsoluteURL(
+          payload.filter.type,
+          new URL(payload.card_id),
+        );
+        if (!isResolvedCodeRef(maybeCodeRef)) {
+          throw new Error('Query returned by ai bot is not fully resolved');
+        }
+        payload.filter.type = maybeCodeRef;
+        let searchCardResource = await getCards({
```

Comment:
let's use the search-resource directly instead of going thru runtime common. the way we can be explicit about who the owner is. the runtime common form of this exists for card-defs to consume specifically. since we are in the host app there is not a need to jump thru this layer

Reply:
Also, instead of creating a  new resource for every search, perhaps we should just use a single resource and modify the thunk for the resource to run a new search. trying to be mindful the of the lessons we learned from Ed' ember resources discussion

Reply:
Good point